leits/MeetingBar
# 🎉feat (fantastical): allow to open calendar events in fantastical

### Status
**IN DEVELOPMENT**

### Description
- we have added a menu item Open in fantastical to the event menu entry besides open in calendar
- the action will trigger the url handler for fantastical with a date and title
- fantastical will automatically open as app and show the appointment

### Steps to Test or Reproduce
- Create fantastical via homebrew (brew install --cask fantastical)
- Configure a calendar in Fantastical 
- Create an event which is available in meetingbar and fantastical
- open the event via "open in fantastical" -> fantastical should show the event

### Questions
- Should we introduce a new configuration option "Use calendar app" with 2 options "macos Calendar" and "Fantastical" so that the user can configure the used app?
- Should we check if fantastical is installed, so that fantastical and the whole configuration is only shown if the app is installed?
